src/drt/src/pa/FlexPA.h

```diff
@@ -593,17 +593,19 @@ class FlexPA
 
   void prepPattern();
 
-  int prepPatternInst(frInst* inst, int curr_unique_inst_idx, double x_weight);
+  void prepPatternInst(frInst* inst);
 
-  int genPatterns(const std::vector<std::pair<frMPin*, frInstTerm*>>& pins,
-                  int curr_unique_inst_idx);
+  int prepPatternInstHelper(frInst* inst, bool use_x = true);
```

Comment:
for better readability, I prefer not having this as a default valued argument. It brings attention to the one reading the code that there is something called use_x in here.
It would be also helpful if we have a documentation for this function stating this parameter as it is not easily interpreted by name. Not necessarily in this PR.